shell/app/node_main.cc

```diff
@@ -132,48 +120,18 @@ int NodeMain(int argc, char* argv[]) {
 
 #if BUILDFLAG(IS_LINUX)
   auto os_env = base::Environment::Create();
-  std::string fd_string;
-  if (os_env->GetVar("CRASHDUMP_SIGNAL_FD", &fd_string)) {
-    base::StringToInt(fd_string, &crashdump_signal_fd);
-    ElectronCrashReporterClient::Create();
-    crash_reporter::InitializeCrashpad(false, "node");
-    crash_keys::SetCrashKeysFromCommandLine(
-        *base::CommandLine::ForCurrentProcess());
-    crash_keys::SetPlatformCrashKey();
+  std::string fd_string, pid_string;
+  if (os_env->GetVar("CRASHDUMP_SIGNAL_FD", &fd_string) &&
+      os_env->GetVar("CRASHPAD_HANDLER_PID", &pid_string)) {
+    int fd, pid = -1;
```

Comment:
```suggestion
    int fd = -1, pid = -1;
```

Reply:
Addressed in [ecd98e8](https://github.com/electron/electron/pull/36460/commits/ecd98e89805fb167b2074a91f3a49e0c4aa68c5a)